### ISSUE TITLE
Do you have a Keras 2 version？

### ISSUE THREAD

**Issue by DerekGrant** [titu1994/DenseNet#21]
This is a very useful project for me, I spent some time to modify some details to make sure it can run on Keras to like add "require_flatten=include_top". However, if you can provide a Keras 2 version that will be better.

**Comment by titu1994** [comment]
But this is a Keras 2 model. Where does it show warnings that it is using Keras 1 format?

**Comment by DerekGrant** [comment]
In Densenet.py line 108 : "include_top=include_top"
but in Keras document 
https://github.com/fchollet/keras/blob/master/keras/applications/imagenet_utils.py
```
def _obtain_input_shape(input_shape,
                        default_size,
                        min_size,
                        data_format,
                        require_flatten,
                        weights=None):
```
Then when I run cifra100.py, met I an error said “"include_top=include_top" is an unexcept parameter. So   I modify it as "require_flatten=include_top" and make it to work.

**Comment by titu1994** [comment]
This was changed in a commit 13 days ago. Thanks for bringing it up. I'll have to take some time to go through all of my model builders for that.

**Comment by titu1994** [comment]
Most probably, scale layers wont be needed. Keras BatchNorm has scaling inbuilt. I will have to port the weights though, which will take time.
